src/components/UI/BookComponent/dummyBooks.ts

```diff
@@ -0,0 +1,19 @@
+import { Book } from "../../../Types/bookTypes";
+
+export const dummyBooks: Book[] = [
```

Comment:
merge the main branch, remove this file, and use the dummy data provided

Reply:
no dummy data for books is provided in dummydata.ts
I'll add this book data to the file ok